scripts/performance/benchmark

```diff
@@ -105,6 +103,17 @@ def run_benchmark(pid, output_file, data_interval):
     return 0
 
 
+def _get_underlying_python_process(process):
+    # For some scripts such as the streaming CLI commands, the process is
+    # nested under a shell script that does not account for the python process.
+    # We want to always be measuring the python process.
+    children = process.children(recursive=True)
+    for child_process in children:
+        if 'python' in child_process.name():
```

Comment:
This should be a case insensitive search.  If you look at my process tree in my original comment I have:

```
 72364 jamesls /usr/local/Cellar/python/2.7.11/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python /usr/local/bin/aws s3 cp - s3://bucket
```